integration/shared/isolated/delete_orphaned_routes_command_test.go

```diff
@@ -57,9 +57,6 @@ var _ = Describe("delete-orphaned-routes command", func() {
 			It("deletes all the orphaned routes", func() {
 				Eventually(helpers.CF("delete-orphaned-routes", "-f")).Should(SatisfyAll(
 					Exit(0),
-					Say("Getting routes as"),
```

Comment:
I'm not seeing any test coverage for the `Deleting routes as <user>` message - please make sure that message is asserted on wherever it's printed.